src/Microsoft.AspNet.Hosting.WindowsServices/WebHostExtensions.cs

```diff
@@ -6,37 +6,37 @@
 namespace Microsoft.AspNet.Hosting.WindowsServices
 {
     /// <summary>
-    ///     Extensions to <see cref="IWebApplication" for hosting inside a Windows service. />
+    ///     Extensions to <see cref="IWebHost" for hosting inside a Windows service. />
     /// </summary>
-    public static class WebApplicationExtensions
+    public static class WebHostExtensions
```

Comment:
WebHostServiceRunnerExtensions
Extension classes can't be directly named ClassFooExtensions because you start running into conflicts when everybody does it.